src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java

```diff
@@ -143,4 +146,40 @@ public static void selectDedupeByFunction(String function) {
     ElementHelper.selectDropdownOption(CdfBigQueryPropertiesLocators.dedupeFunctionName,
                                        CdfPluginPropertiesLocators.locateDropdownListItem(function));
   }
+
+  /**
+   * Enters a value into the Plugin Property (input) element.
+   *
+   * @param pluginProperty The value of the @data-cy attribute associated with the Plugin Property.
+   *                       If the pluginProperty is present in, then its @data-cy attribute value is fetched from it;
+   *                       otherwise, the pluginProperty is used as is.
+   * @param value          The value to be entered into the Plugin Property input element.
+   */
+  public static void enterValueInInputProperty(String pluginProperty, String value) {
+    WebElement pluginPropertyInput = getInputPluginPropertyElement(pluginProperty);
+
+    String valueFromPluginPropertiesFile = PluginPropertyUtils.pluginProp(value);
+    if (valueFromPluginPropertiesFile == null) {
+      ElementHelper.sendKeys(pluginPropertyInput, value);
+      return;
+    }
+
+    ElementHelper.sendKeys(pluginPropertyInput, valueFromPluginPropertiesFile);
+  }
+
+  /**
+   * Get Plugin Property input webelement
+   *
+   * @param pluginProperty @data-cy attribute value of Plugin Property.
```

Comment:
The decription of pluginProperty is different in line 153 and 173 . EIther we have to make same comments or delete one of them .